### DISCUSSION TITLE
Github education application denied

### DISCUSSION THREAD

**Discussion by hyoon14** [community/community#134620] [GitHub Education]
### Hi everyone,

Hi everyone, I am enrolled in a Masters program that I began in person, then attained a job outside of the state, by which the program allowed me to re-enroll as a remote student. I am having trouble verifying this, can someone here help me with this to get approved for github education?

**Comment by Aryankpoor** [comment]
Hi, the acceptane of your github education application does not only depend upon whether you are currently a student or not. There are a lot of factors.
-  It is recommended that you update your socials and github to display your pictures, keep your instagram account from private to public as they review it also. 
- Yes being a student matters, your instagram and github should contain some sort of proof that you are indeed a student. Provide them with a nicely setup linkedIn profile while you are at it. 
- Also register with the email address provided by any educational institution if you are going to if you haven't already. 
All of this helped me get the github student developer pack. Let me know if it works for you.

**Reply by samus-aran** [reply]
Hi @Aryankpoor 👀 

> - It is recommended that you update your socials and github to display your pictures, keep your instagram account from private to public as they review it also.
> - Yes being a student matters, your instagram and github should contain some sort of proof that you are indeed a student. Provide them with a nicely setup linkedIn profile while you are at it.

No we **do not** review an applicants' social media. If you were successful after making those changes then that is coincidental as we only use information the applicant has provided and your GitHub details.

**Comment by Aryankpoor** [comment]
Might be something else, because I got rejected in my first couple of tries and in my last application before I got accepted, I had only changed my instagram from private to public. If that was not the reason then I still dont understand what changed between my last rejected application and the application which was accepted.

**Comment by PakmanGames** [comment]
Try uploading a proof of enrollment letter with a valid date or your student ID if there's a date on it. And make sure your name in your account's billing settings is the same as the name on the enrollment letter or student ID.
